hack/lib/build/binaries.sh

```diff
@@ -219,6 +219,50 @@ os::build::internal::build_binaries() {
         local_ldflags+=" -s"
       fi
 
+      #Add Windows File Properties/Version Info and Icon Resource for oc.exe
+      if [[ "$platform" == "windows/amd64" ]]; then
+        os::build::version::get_vars
+        os::util::ensure::gopath_binary_exists 'goversioninfo'
+        local major="${OS_GIT_MAJOR}"
```

Comment:
Are we not using these `major` and `minor` anymore? Seemed reasonable to use them, and you could update `version.sh` to also get you `patch` as well.